src/components/FlawEditFormOld.vue

```diff
@@ -0,0 +1,280 @@
+<script setup lang="ts">
```

Comment:
Instead of adding "old" shouldn't we continue to rely on git commit tracking?

Reply:
These were left in the repo for convenience of referring to recently-old rewritten code. Now removing all the old stuff.